lib/config/definitions.ts

```diff
@@ -148,6 +148,17 @@ const options: RenovateOptions[] = [
     allowString: true,
     cli: false,
   },
+  {
+    name: 'migratePresets',
+    description:
+      'Define presets here which have been removed or renamed and should be migrated automatically.',
+    type: 'object',
+    admin: true,
+    default: {},
+    additionalProperties: {
+      type: 'string',
+    },
```

Comment:
Is this for schema generation? Can we use it for validation simplification?

Reply:
🤷 maybe.. but prefer not in this PR